src/main/java/org/elmlang/intellijplugin/Elm.bnf

```diff
@@ -84,11 +91,12 @@ as_clause ::= AS upper_case_id
 
 exposing_clause ::= EXPOSING exposed_values { methods = [isExposingAll getReferencesStream] }
 
-comment ::=
-    LINE_COMMENT
-    | level_comment
+line_comment ::= LINE_COMMENT_LEX
```

Comment:
Why the `_LEX` suffix? It doesn't match the rest of the token names coming from Flex.